guides/plugins/plugins/administration/adding-directives.md

```diff
@@ -0,0 +1,58 @@
+# Adding directives
+
+## Overview
+
+Directives in the Shopware 6 Administration are essentially as in any other Vue application.
+This guide will teach you how to register your directives on a global and on a local scope.
+
+Learn more about Vue Directives in their [documentation](https://vuejs.org/v2/guide/custom-directive.html)
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance and full access to both the files and preferably a registered module. Of course you'll have to understand JavaScript, but that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Registering a directives globally
+
+Directives can be globally registered via the [Shopware Objects](./the-shopware-object.md) `register` helper function as seen below:
+
+{% code title="<plugin-root>/src/Resources/app/administration/app/src/directive/focus.js" %}
+```javascript
+const { Directive } = Shopware;
+
+Directive.register('focus', {
+    // When the bound element is inserted into the DOM...
+    inserted: function (el) {
+      // Focus the element
+      el.focus()
+    }
+  })
+```
+{% endcode %}
+
+As you might have seen, this is the exact same example as in the [Vue documentation](https://vuejs.org/v2/guide/custom-directive.html).
+The only thing that's left now is to import this file in your `main.js` and then you can use it as you would a normal Vue directive.
```

Comment:
maybe you should give a small example how to use this directive `focus` in a new component / template

Reply:
Well it's exactly the same as in any other Vue application and i link to the Vue docs and say this multiple times. So i think this should suffice

Reply:
It's more comfortable for the user to not being forced to change between various articles though. So I'd like an example as well, after all.